### PR TITLE
DVCSMP-2043 OSRAM: Saturation values are not updated in older firmwares

### PR DESCRIPTION
**Pull request by larsfinander**

### CODE REVIEW

File: devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy [hunk 1]
```diff
@@ -177,5 +177,5 @@ def setHue(value) {
 
 def setSaturation(value) {
     def scaledSatValue = zigbee.convertToHexString(Math.round(value * 0xfe / 100.0), 2)
-    zigbee.command(COLOR_CONTROL_CLUSTER, SATURATION_COMMAND, scaledSatValue, "0500")      //payload-> sat value, transition time
+    zigbee.command(COLOR_CONTROL_CLUSTER, SATURATION_COMMAND, scaledSatValue, "0500") + "delay 300" + zigbee.readAttribute(0x0300, ATTRIBUTE_SATURATION)
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: The transition time used for the Move to Saturation command is 0x0005 which translates to 1/2s so the delay before reading the Saturation attribute should be at least 500ms.

Comment (larsfinander) [reply devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy]: THanks, done!

**Comment by larsfinander** [comment]
@workingmonk @tpmanley Review please, I was considering trying to add code to detect firmware and only do this for older ones but not sure if it is worth it. My first attempt at reading firmware didnt succeed and I am not sure if it is exposed to DTH.

**Review by tpmanley — APPROVED** [review]
For consistency should use `COLOR_CONTROL_CLUSTER` instead of hardcoding `0x0300` but otherwise looks good

**Pull request merged**